meta-mender-core/recipes-mender/mender-client/mender-client_3.2.0.bb

```diff
@@ -0,0 +1,31 @@
+require mender-client.inc
+
+################################################################################
+#-------------------------------------------------------------------------------
+# THINGS TO CONSIDER FOR EACH RELEASE:
+# - SRC_URI (particularly "branch")
+# - SRCREV
+# - DEFAULT_PREFERENCE
+#-------------------------------------------------------------------------------
+
+SRC_URI = "git://github.com/mendersoftware/mender;protocol=https;branch=3.2.x"
+
+# Tag: 3.2.0
+SRCREV = "f6e9401329ff26e27955651a20f9025275c717bf"
+
+# Enable this in Betas, and in branches that cannot carry this major version as
+# default.
+# Downprioritize this recipe in version selections.
+# DEFAULT_PREFERENCE = "-1"
```

Comment:
Needs to be down-priotized like the rest of the `mender-client` > 3.0 recipes in `dunfell`